src/ilmm.jl

```diff
@@ -43,7 +46,7 @@ Return the diagonal element of Σ.
 ```jldoctest
 julia> Σ = Diagonal(Fill(2, 3));
 
-julia> noise_var(Σ) == 2
+julia> LinearMixingModels.noise_var(Σ) == 2
```

Comment:
My advice would be against using internal details in the docstrings. In general, stuff that users shouldn't have to know about probably shouldn't be in the docstrings. So things like this and `reshape_y` should probably just be tested in the tests.

Reply:
Done